test/python/tensorflow/python_tests_list.txt

```diff
@@ -112,6 +112,35 @@ conv_ops_test.Conv2DTest.testInputGradientValidPaddingStrideTwo
 #conv_ops_test.Conv2DTest.testFilterGradientValidPaddingStrideTwo 
 #conv_ops_test.Conv2DTest.testOpEdgeCases
 
+conv_ops_3d_test.Conv3DTest.testConv3D1x1x1Filter
+conv_ops_3d_test.Conv3DTest.testConv3D1x1x1Filter2x1x1Dilation
+conv_ops_3d_test.Conv3DTest.testConv3D2x2Depth3ValidBackpropFilterStride1x1Dilation2x1 
+conv_ops_3d_test.Conv3DTest.testConv3D2x2Depth3ValidBackpropInputStride1x1Dilation2x1
+conv_ops_3d_test.Conv3DTest.testConv3D2x2x2Filter 
+conv_ops_3d_test.Conv3DTest.testConv3D2x2x2Filter1x2x1Dilation
+conv_ops_3d_test.Conv3DTest.testConv3D2x2x2FilterStride2 
+conv_ops_3d_test.Conv3DTest.testConv3D2x2x2FilterStride2Same 
+conv_ops_3d_test.Conv3DTest.testConv3DStride3 
+conv_ops_3d_test.Conv3DTest.testConv3DStrides 
+conv_ops_3d_test.Conv3DTest.testFilterGradientKernelSizeMatchesInputSize 
+conv_ops_3d_test.Conv3DTest.testFilterGradientSamePaddingStrideOne 
+conv_ops_3d_test.Conv3DTest.testFilterGradientSamePaddingStrideThree 
+conv_ops_3d_test.Conv3DTest.testFilterGradientSamePaddingStrideTwo 
+conv_ops_3d_test.Conv3DTest.testFilterGradientValidPaddingStrideOne 
+conv_ops_3d_test.Conv3DTest.testFilterGradientValidPaddingStrideThree 
+conv_ops_3d_test.Conv3DTest.testFilterGradientValidPaddingStrideTwo 
+conv_ops_3d_test.Conv3DTest.testInputGradientKernelSizeMatchesInputSize 
+conv_ops_3d_test.Conv3DTest.testInputGradientSamePaddingDifferentStrides 
+conv_ops_3d_test.Conv3DTest.testInputGradientSamePaddingStrideOne 
+conv_ops_3d_test.Conv3DTest.testInputGradientSamePaddingStrideThree 
+conv_ops_3d_test.Conv3DTest.testInputGradientSamePaddingStrideTwo 
+conv_ops_3d_test.Conv3DTest.testInputGradientValidPaddingStrideOne 
+conv_ops_3d_test.Conv3DTest.testInputGradientValidPaddingStrideThree
+conv_ops_3d_test.Conv3DTest.testInputGradientValidPaddingStrideTwo 
+conv_ops_3d_test.Conv3DTest.testKernelSizeMatchesInputSize 
+#conv_ops_3d_test.Conv3DTest.testKernelSmallerThanStride 
+conv_ops_3d_test.Conv3DTest.test_session 
+
 conv_ops_test.DepthwiseConv2DTest.testConv2D2x2Filter
```

Comment:
Could you also add the test from `conv3d_transpose_test` which tests Conv3D?

Reply:
Sure, added now.